src/lumigo_opentelemetry/instrumentations/django/tested_versions/3.12/django

```diff
@@ -0,0 +1,93 @@
+3.2.1
```

Comment:
How did you create this versions file? did you run the versions check locally to create it?